algorithms/kernel/objective_function/mse/mse_dense_default_batch_impl.i

```diff
@@ -628,6 +637,18 @@ inline services::Status MSEKernel<algorithmFPType, method, cpu>::compute(Numeric
                     }
                 }
             }
+            if (l2)
```

Comment:
checking float without eps

Reply:
it's ok here, this parameter is directly set by user (default value is existed too)